packages/xo-web/src/xo-app/sr/tab-disks.js

```diff
@@ -173,18 +183,125 @@ const FILTERS = {
 
 // ===================================================================
 
+@injectIntl
+@propTypes({
+  onClose: propTypes.func,
+  sr: propTypes.object.isRequired,
+})
+class NewDisk extends Component {
+  _createDisk = () => {
+    const { sr, onClose = noop } = this.props
+    const { name, readOnly, size } = this.state
+
+    return createDisk(name, size, sr, {
+      mode: readOnly ? 'RO' : 'RW',
+    }).then(onClose)
+  }
+
+  render () {
+    const { formatMessage } = this.props.intl
+    const { name, readOnly, size } = this.state
+
+    return (
+      <form id='newDiskForm'>
+        <fieldset className='form-inline'>
```

Comment:
Do you need the `fieldset` if there's only one? What about putting the `form-inline` on the form?